src/pages/Farm/FarmOverview.tsx

```diff
@@ -29,17 +29,16 @@ const TokenGroup = styled("div")(() => ({
 
 export default function FarmOverview({
   farmName,
-  poolName,
+  poolTokens,
   aprs,
   tvl,
   myStake,
   onClickStake,
 }: // onClickClaim,
 FarmOverviewProps): JSX.Element {
   const { t } = useTranslation()
-  const [farmData] = usePoolData(poolName || "")
-  const farmTokens = farmData.tokens
-
+  const { chainId } = useActiveWeb3React()
+  if (!chainId) return <div></div>
```

Comment:
Can you add extra newline after this for style purposes? You can also return `null`

Reply:
BTW, getTokenByAddress is return `Token||null` so after filtering we will get null again so we need to check again.